components/cyclebutton/index.css

```diff
@@ -11,9 +11,6 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-CycleButton {
-  /* Be square */
-  padding-block: 0;
-  padding-inline: var(--spectrum-actionbutton-m-icononly-padding-left) var(--spectrum-actionbutton-m-icononly-padding-right);
```

Comment:
These values seem to be redundant—removing them had no effect on the Cycle Button's squareness and the `padding-inline` removal did not affect the positioning of the Cycle Button's icon.

I added `hover` and `focus` stories to Storybook to cover this, and compared with what's on production today.